source/features/split-issue-pr-search-results.tsx

```diff
@@ -12,20 +12,51 @@ function cleanLinks(): void {
 	}
 }
 
+function getPageSearchQuery(): SearchQuery {
+	const searchParameters = new URLSearchParams(location.search);
+	return new SearchQuery(searchParameters);
+}
+
 type GitHubConversationType = 'pr' | 'issue';
 
 function createUrl(type: GitHubConversationType, pathname = location.pathname): string {
 	const url = new URL(pathname, location.origin);
-	url.searchParams.set('q', pageSearchQuery.get() + ` is:${type}`);
+	const searchQuery = getPageSearchQuery();
+	// Remove all existing is:pr/issue query parts, only the last is used anyways
+	searchQuery.remove('is:pr', 'is:issue');
+	searchQuery.add(`is:${type}`);
+	url.searchParams.set('q', searchQuery.get());
 	url.searchParams.set('type', 'Issues');
+
+	// Copy the language (l) and state (closed/open) search parameters if they are set
+	for (const name of ['l', 'state']) {
+		if (searchQuery.searchParams.has(name)) {
+			url.searchParams.set(name, searchQuery.searchParams.get(name)!);
+		}
+	}
```

Comment:
Instead of copying them, you can set the whole search at once.

Reply:
My main thought here was to only copy things that are actually available/used.
E.g. this would avoid copying something like [`?package_type=npm`](https://github.com/search?package_type=npm&q=refined+github&type=RegistryPackages) which has no effect.
But if it is more readable I can also just copy all search params instead.